app/components/molecules/perk-info/perk-info.styles.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const LeadingIcon = styled.img`
+  width: 50px;
+  margin-bottom: 5px;
+`;
+
+export const Section = styled.div`
+  border-top: 4px solid ${({theme}) => theme.colors.black};
+  padding-top: 5px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: left;
+  max-width: 85px;
+`;
+
+export const Label = styled.h2`
```

Comment:
Should this be an h2? I think it should be some other tag since we will be using h2 for the `TitleSection` component.